systemd/voms@.service.d/after-mariadb.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
+Description=Ensures that VOMS starts after MySQL db is loaded
```

Comment:
You cannot put a `Description` here, that will override the one in the voms service itself.

Reply:
You're absolutely right, fixed...

Reply:
New file looks good to me!